packages/theming/default-theme/src/defaultTheme.ts

```diff
@@ -77,29 +77,37 @@ export function defaultSpacing(): Spacing {
   return { s2: '4px', s1: '8px', m: '16px', l1: '20px', l2: '32px' };
 }
 
-export const defaultFluentTheme: Theme = {
-  colors: getStockWebPalette(),
-  typography: _defaultTypography(),
-  spacing: defaultSpacing(),
-  shadows: createShadowAliasTokens('light'),
-  components: {},
-  host: { appearance: 'light' },
-};
+export function defaultFluentTheme(): Theme {
```

Comment:
The issue with this is that this will return a new object every time it's called. The main drawbacks of generating a new object on every call is that it can trigger unnecessary rerenders since the instance is now different, and it will redo work that doesn't necessarily need to be done when the function is called.

You might be able to avoid creating a const by memoizing the function instead, and then we don't have the issues above?

Reply:
Yes actually did consider that at some point but forgot about it, will do & thanks for reminding me